docs/tutorials/python/schema_operations.md

```diff
@@ -0,0 +1,59 @@
+JSON Schema is a tool used to validate data. In Synapse, JSON Schemas can be used to validate the metadata applied to an entity such as project, file, folder, table, or view, including the [annotations](https://help.synapse.org/docs/Annotating-Data-With-Metadata.2667708522.html) applied to it. To learn more about JSON Schemas, check out [JSON-Schema.org](https://json-schema.org/).
+
+Synapse supports a subset of features from [json-schema-draft-07](https://json-schema.org/draft-07). To see the list of features currently supported, see the [JSON Schema object definition](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/schema/JsonSchema.html) from Synapse's REST API Documentation.
+
+In this tutorial, you will learn how to create these JSON Schema using an existing data-model.
+
+## Tutorial Purpose
+You will create a JSON schema using your data model.
+
+## Prerequisites
+* You have a working [installation](../installation.md) of the Synapse Python Client.
+* You have a data-model, see [here](https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2473623559/The+Data+Model+Schema)
+
+## 1. Imports
+
+```python
+{!docs/tutorials/python/tutorial_scripts/schema_operations.py!lines=1-2}
+```
+
+## 2. Set up your variables
+
+```python
+{!docs/tutorials/python/tutorial_scripts/schema_operations.py!lines=4-10}
+```
+
+To create a JSON Schema you need a data-model, and the data-types you want to create.
+The data-model must be in either CSV or JSON-LD form. The data model maybe a local path or a URL.
+
+The data-types must exist in your data-model. This cna be a list of data-types, or `None` to create all datatatypes in the data-model.
+
+## 3. Log into Synapse
+```python
+{!docs/tutorials/python/tutorial_scripts/schema_operations.py!lines=12-13}
+```
+
+
+## 4. Create the JSON Schema
+Create the JSON Schema(s)
+```python
+{!docs/tutorials/python/tutorial_scripts/schema_operations.py!lines=15-23}
+```
+You should see the first JSON Schema for the datatype(s) you selected printed.
```

Comment:
Could you include the example output of what would be printed here given the example URL data model that we will use? That way folks have a bit of context of what to expect.